src/views/RoomDetail.vue

```diff
@@ -105,16 +117,8 @@ export default {
           })
         })
     },
-    fetchUserIDList: async function () {
-      if (this.room.checkIns.user) {
-        this.room.checkIns.user.forEach(user => this.userIdList.todayIDList.push(user.id))
-      }
-      if (this.room.checkInsTomorrow.user) {
-        this.room.checkInsTomorrow.user.forEach(user => this.userIdList.tomorrowIDList.push(user.id))
-      }
-    },
     protection: async function () {
-      await this.signInProtection(this.currentDate, this.room.checkIns.user)
+      await this.signInProtection(this.currentDate, this.room.checkIns)
```

Comment:
Wieso wird hier nicht mehr weiter über das User Objekt gegangen, aber dafür noch eine Zeile weiter?

Reply:
Weil in CheckIn jetzt die User direkt drin liegen, da ich mir von der Referenz, die davor drin war mir die User aus der User Tabelle hole um die Namen anzuzeigen.